setup.cfg

```diff
@@ -48,6 +48,13 @@ omit=
 source = src/prefect
 parallel = False
 
+[coverage:report]
+exclude_lines =
+    def __repr__
```

Comment:
I actually would prefer we _do_ test `reprs`.  I once caught a recursion error due to a repr that I wouldn't have otherwise caught.  Additionally, [reprs are important to us](#617 )

Open to pushback on that though

Reply:
Good call!